api/tests/python/end_points/test_namex_search.py

```diff
@@ -591,35 +591,44 @@ def test_namex_search_submitted_start_and_end_date_invalid_date_format(client,
         assert 'Must be of date format %Y-%m-%d' in msg
 
 
-def test_namex_search_direct_nrs_bad_roles(client, jwt, app):
-    """Test searching directly using name request numbers with bad roles."""
-    base_nrs = generate_nrs(5, [], [], [])
-    for nr in base_nrs:
-        nr.save_to_db()
-    qs = "&".join(["nrNumbers="+nr.nrNum for nr in base_nrs])
-    rv = client.get(
-        f'api/v1/requests?{qs}',
-        headers=create_header(jwt, [])
-    )
-    assert rv
-    assert rv.status_code == 403
-
-
-def test_namex_search_direct_nrs(client, jwt, app):
+@pytest.mark.parametrize('identifiers, search_identifier, status, name, total_results', [
+    (['NR 0', 'NR 1', 'NR 2', 'NR 3', 'NR 4'], '', State.APPROVED, '', 1),
+    (['NR 0', 'NR 1', 'NR 2', 'NR 3', 'NR 4'], '', State.DRAFT, '', 1),
+    (['NR 0', 'NR 1', 'NR 2', 'NR 3', 'NR 4'], '', '', 'TEST', 5),
+    (['NR 0', 'NR 1', 'NR 2', 'NR 3', 'NR 4'], '', '', 'BAD', 0),
+    (['NR 0', 'NR 1', 'NR 2', 'NR 3', 'NR 4'], 'NR 1', '', '', 1),
+    ([], '', '', '', 0),
+    (['NR 0', 'NR 1', 'NR 2', 'NR 3', 'NR 4'], '', '', '', 5),
+])
+def test_namex_search_direct_nrs(client, jwt, app, identifiers, search_identifier, status, name, total_results):
```

Comment:
I find it pretty helpful to have the name of each test as the first arg in each param tuple. That way if one case fails its really easy for everyone to know the scenario they broke

Reply:
oops didn't mean to approve the draft haha

Reply:
Great suggestion, I'll fix this